Samplero/Samplero/Screen/TermsAndConditions/ViewModel/TermsViewModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  TermsViewModel.swift
+//  Samplero
+//
+//  Created by Minkyeong Ko on 2022/10/13.
+//
+
+import Foundation
+
+import RxSwift
+import RxCocoa
```

Comment:
p5.
알파벳 순으로 하기로 했습니다

```suggestion
import RxCocoa
import RxSwift
```

Reply:
p5.
그런적 없습니다.